lib/cache/hash/hash.ts

```diff
@@ -0,0 +1,9 @@
+import * as hash from 'object-hash';
+
+export class HashService {
+
+  public hash(value: any): string {
```

Comment:
To avoid constructions of the form `hash.hash()` I'd suggest that we rename the method to `.calculate()`

Reply:
And the `HashService` seems to be a strange in the `cache` domain. I'd suggest moving it to some kind of utils outside of `cache`.